lazy_tensor_core/lazy_tensor_core/csrc/tensor_ops.cpp

```diff
@@ -1,7 +1,9 @@
 #include "lazy_tensor_core/csrc/tensor_ops.h"
 
+#include "lazy_tensor_core/csrc/aten_ltc_bridge.h"
 #include "lazy_tensor_core/csrc/helpers.h"
 #include "lazy_tensor_core/csrc/ir.h"
+#include "lazy_tensor_core/csrc/ts_backend/LazyLazyIr.h"
```

Comment:
Is this .h naming only temporary?

Reply:
it got generated from the pattern {backend_key}LazyIr.h. 

It happens that our 2 backends have the key 'xla' and 'lazy' for now. 

I'm not sure what we're gonna do for naming of backends later on, so I didn't think about this too much yet.

perhaps for now, I can just remove the second 'Lazy' so we get 'LazyIr' and 'XlaIr', but, i am not sure if that is actually clearer or might be confusing to XLA folks who actually have their own IR